interface/modules/zend_modules/module/Application/src/Application/Plugin/Phimail.php

```diff
@@ -32,12 +33,13 @@ class Phimail extends AbstractPlugin
   /**
   *
   * Application Table Object
-  * Listener Oblect
-  * @param type $sm Service Manager
+  * Listener Object
+  * @param type $container ContainerInterface
   */
-    public function __construct($sm)
+    public function __construct(ContainerInterface $container)
     {
-        $sm->get('Zend\Db\Adapter\Adapter');
+        // TODO: again why grab the service... construct the tables and do nothing with them.  Can this code be removed?
```

Comment:
You are now the openemr Zend expert, so if you think it can be removed, then it can be removed :)